client/state/plans/test/selectors.js

```diff
@@ -51,94 +51,21 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '#getPlanPriceObject()', () => {
-		it( 'should return price plan object', () => {
+	describe( '#getPlanRawPrice()', () => {
+		it( 'should return annual raw price', () => {
 			const state = getStateInstance();
-			const priceObject = getPlanPriceObject( state, 1003 );
-			expect( priceObject ).to.eql( {
-				currencySymbol: '$',
-				decimalMark: '.',
-				dollars: 99,
-				cents: 0
-			} );
+			const price = getPlanRawPrice( state, 1003 );
+			expect( price ).to.eql( 99 );
```

Comment:
It's the `to.eql( 99 )` part that I'm asking about. Where did that come from? It doesn't appear to be setup here in the test, so I assume it comes from the outside world (the API) but since I don't see a call to get it I assume there's a mental bridge keeping those equal - one that's implicit and easily forgotten.